### ISSUE TITLE
[Feature Request] multiple connections

### ISSUE THREAD

**Issue by rsd3k** [rg3/youtube-dl#3747]
Is there a way to enable multiple connections for download. Some download clients (eg aria2, Jdownloader) have the feature of specifying the number of connections.

**Comment by steebchen** [comment]
Yeah that would be really great for sites like nowvideo.li.

**Comment by yan12125** [comment]
FYI: aria2 is supported as a external downloader. For example:
```
youtube-dl --external-downloader aria2c --external-downloader-args "--options --for --aria2c"
```

**Comment by steebchen** [comment]
Thank you, that's great.

**Comment by yan12125** [comment]
Closed as duplicate of #350

**Comment by yan12125** [comment]
Sorry, #350 is somewhat different. Follow #1498 instead.
